dpdispatcher/dp_cloud_server.py

```diff
@@ -180,25 +182,28 @@ def map_dp_job_state(status):
         if isinstance(status, JobStatus):
             return status
         map_dict = {
-            -1:JobStatus.terminated,
-            0:JobStatus.waiting,
-            1:JobStatus.running,
-            2:JobStatus.finished,
-            3:JobStatus.waiting,
-            4:JobStatus.running,
-            5:JobStatus.terminated,
-            6:JobStatus.running,
+            -1: JobStatus.terminated,
+            0: JobStatus.waiting,
+            1: JobStatus.running,
+            2: JobStatus.finished,
+            3: JobStatus.waiting,
+            4: JobStatus.running,
+            5: JobStatus.terminated,
+            6: JobStatus.running,
         }
         if status not in map_dict:
             dlog.error(f'unknown job status {status}')
             return JobStatus.unknown
         return map_dict[status]
 
-
     # def check_finish_tag(self, job):
     #     job_tag_finished = job.job_hash + '_job_tag_finished'
     #     return self.context.check_file_exists(job_tag_finished)
 
 
 class Lebesgue(DpCloudServer):
     pass
+
+
+class Bohrium(DpCloudServer):
```

Comment:
This will make the parameters duplicated. We can provide an `alias` attribute instead...

Reply:
I have remove the Bohrium since Lebesgue is enough for use